Tests/SwiftLintFrameworkTests/Resources/ProjectMock/ChildConfig/Test1/Main/main.yml

```diff
@@ -9,3 +9,4 @@ excluded:
 line_length: 80
 
 child_config: child1.yml
+parent_config: nonExistingParent.yml
```

Comment:
Would be nice if something was asserted in a test as well.

Reply:
`testValidChildConfig` validates it already. It fails to load the config with a non existent parent and fails. After the change is applied it succeeds as expected.

Let me know if you want some specific test scenario covered, though I can't think of one other than successfully loading the config 🤷‍♂️